File: Test_Coverage/processed_dataset/tests/test_HumanEval_147.py
from functions.HumanEval_147 import get_max_triples

def test_HumanEval_147():
 
     assert get_max_triples(5) == 1
     assert get_max_triples(6) == 4
     assert get_max_triples(10) == 36
     assert get_max_triples(100) == 53361

File: Test_Coverage_Java/src/test/java/test_HumanEval_147.java
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class HumanEvalTest {

    @Test
    public void test_HumanEval_147() {
        int maxTriples = HumanEval_147.getMaxTriples(5);
        assertEquals(1, maxTriples);

        maxTriples = HumanEval_147.getMAXTRIPLES(6);
        assertEquals(4, maxTriples);

        maxTriples = HumanEval_147.getMAXTRIPLES(10);
        assertEquals(36, maxTriples);

        maxTriples = HumanEval_147.getMAXTRIPLES(100);
        assertEquals(53361, maxTriples);
    }
}